lutra/Taskfile.yml

```diff
@@ -0,0 +1,57 @@
+version: "3"
+
+vars:
+  packages: -p lutra
+
+tasks:
+  run:
+    cmds:
+      - cmd: cargo run -- print example-project
+
+  fmt:
+    desc: Format prqlc source files
+    cmds:
+      - cmd: |
+          # remove trailing whitespace
+          rg '\s+$' --files-with-matches --glob '!*.{rs,snap}' . \
+          | xargs -I _ sh -c "echo Removing trailing whitespace from _ && sd '[\t ]+$' '' _"
+
+      - cmd: |
+          # rustfmt
+          cargo fmt {{.packages}}
+
+      - cmd: |
+          # no dbg
+          rg 'dbg!' --glob '*.rs' . --no-heading
+        ignore_error: true
+
+  test-fast:
+    desc: A fast test used for feedback during development
+    cmds:
+      - cmd: |
+          INSTA_FORCE_PASS=1 cargo nextest run {{.packages}} --no-fail-fast
+
+      - cmd: cargo insta review
+
+      - cmd: cargo clippy --all-targets {{.packages}}
+
+  test:
+    desc: The full test. Generates coverage report.
+    env:
+      # Use a different target dir so we don't poison the cache
+      CARGO_LLVM_COV_TARGET_DIR: ../target-cov
+    cmds:
+      - cmd: |
+          cargo \
+            llvm-cov --lcov --output-path lcov.info \
+            nextest \
+            {{.packages}}
```

Comment:
FYI if you want to put `nextest` & `{{.packages}}` into the [base task](https://github.com/PRQL/prql/blob/d52312db53c6f8426594b09a1c90bc8c1b683a83/Taskfile.yml#L339) totally fine w me. Then we have a single task and can pass `packages`.

(but also zero stress if you prefer to have them separate)

Reply:
I like to have a separate task that builds and tests only the thing that I'm currently developing.

What do you mean "pass packages"? To have the base task take packages as argument?

Reply:
> What do you mean "pass packages"? To have the base task take packages as argument?

Currently this task has a `{{.packages}}` arg, so it's possible to pass a subset of packages

> I like to have a separate task that builds and tests only the thing that I'm currently developing.

Very much agree! TBC, this task is almost identical to the task in the parent task, but passes `-p lutra` to `.packages`. So they could be combined.

(I can make a PR later, maybe that's an easier way of explaining)